examples/utils/simulator_example.py

```diff
@@ -0,0 +1,131 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Brain Simulator example script
+
+Example script to generate a run of a participant's data. This generates
+data representing a pair of conditions that are then combined
+
+ Authors: Cameron Ellis (Princeton) 2016
+"""
+import logging
+
+import numpy as np
+from brainiak.utils import simulator as sim
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D as temp
+
+logger = logging.getLogger(__name__)
+
+temp.__module__  # Do in order to remove the pep8 error generated above
```

Comment:
There is a standard way to ignore Flake8 errors:
http://flake8.pycqa.org/en/latest/user/ignoring-errors.html
In this case, the import line should be:
```from mpl_toolkits.mplot3d import Axes3D  # noqa: F401```